<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>Game</string>
    </property>
    <addaction name="actionStart_game"/>
    <addaction name="actionJoin_game"/>
    <addaction name="separator"/>
    <addaction name="actionRestart_game"/>
    <addaction name="actionReturn_main"/>
    <addaction name="action_Pause"/>
    <addaction name="actionReplay"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionShow_Hide_menu"/>
    <addaction name="actionFullscreen"/>
    <addaction name="actionMinimize_to_system_tray"/>
    <addaction name="separator"/>
    <addaction name="actionCard_style"/>
    <addaction name="actionGeneral_style"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionConfigure"/>
    <addaction name="actionGeneral_overview"/>
    <addaction name="actionCard_overview"/>
    <addaction name="actionScenario_overview"/>
    <addaction name="actionView_ban_list"/>
    <addaction name="actionRole_assign_table"/>
    <addaction name="separator"/>
    <addaction name="actionFile_encryption_decryption"/>
    <addaction name="actionBroadcast"/>
    <addaction name="actionAI_Melee"/>
    <addaction name="actionReplay_file_convert"/>
    <addaction name="actionCheck_resource"/>
    <addaction name="separator"/>
    <addaction name="actionEnable_hotkey"/>
    <addaction name="actionExpand_dashboard"/>
    <addaction name="actionView_Discarded"/>
    <addaction name="actionView_distance"/>
    <addaction name="actionServer_information"/>
    <addaction name="actionSurrender"/>
    <addaction name="actionKick"/>
    <addaction name="actionSave_battle_record"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuGame"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionStart_game">
   <property name="text">
    <string>Start game</string>
   </property>
  </action>
  <action name="actionJoin_game">
   <property name="text">
    <string>Join game</string>
   </property>
  </action>
  <action name="actionRestart_game">
   <property name="text">
    <string>Restart game</string>
   </property>
  </action>
  <action name="actionReturn_main">
   <property name="text">
    <string>Return main</string>
   </property>
  </action>
  <action name="action_Pause">
   <property name="text">
    <string>&amp;Pause</string>
   </property>
  </action>
  <action name="actionReplay">
   <property name="text">
    <string>Replay</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionShow_Hide_menu">
   <property name="text">
    <string>Show/Hide menu</string>
   </property>
  </action>
  <action name="actionFullscreen">
   <property name="text">
    <string>Fullscreen</string>
   </property>
  </action>
  <action name="actionMinimize_to_system_tray">
   <property name="text">
    <string>Minimize to system tray</string>
   </property>
  </action>
  <action name="actionCard_style">
   <property name="text">
    <string>Card style</string>
   </property>
  </action>
  <action name="actionGeneral_style">
   <property name="text">
    <string>General style</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="text">
    <string>Configure</string>
   </property>
  </action>
  <action name="actionGeneral_overview">
   <property name="text">
    <string>General overview</string>
   </property>
  </action>
  <action name="actionCard_overview">
   <property name="text">
    <string>Card overview</string>
   </property>
  </action>
  <action name="actionScenario_overview">
   <property name="text">
    <string>Scenario overview</string>
   </property>
  </action>
  <action name="actionView_ban_list">
   <property name="text">
    <string>View ban list</string>
   </property>
  </action>
  <action name="actionRole_assign_table">
   <property name="text">
    <string>Role assign table</string>
   </property>
  </action>
  <action name="actionFile_encryption_decryption">
   <property name="text">
    <string>File encryption/decryption ...</string>
   </property>
  </action>
  <action name="actionBroadcast">
   <property name="text">
    <string>Broadcast ...</string>
   </property>
  </action>
  <action name="actionAI_Melee">
   <property name="text">
    <string>AI Melee</string>
   </property>
  </action>
  <action name="actionReplay_file_convert">
   <property name="text">
    <string>Replay file convert ...</string>
   </property>
  </action>
  <action name="actionCheck_resource">
   <property name="text">
    <string>Check resource</string>
   </property>
  </action>
  <action name="actionEnable_hotkey">
   <property name="text">
    <string>Enable hotkey</string>
   </property>
  </action>
  <action name="actionExpand_dashboard">
   <property name="text">
    <string>Expand dashboard</string>
   </property>
  </action>
  <action name="actionView_Discarded">
   <property name="text">
    <string>View Discarded</string>
   </property>
  </action>
  <action name="actionView_distance">
   <property name="text">
    <string>View distance</string>
   </property>
  </action>
  <action name="actionServer_information">
   <property name="text">
    <string>Server information</string>
   </property>
  </action>
  <action name="actionSurrender">
   <property name="text">
    <string>Surrender</string>
   </property>
  </action>
  <action name="actionKick">
   <property name="text">
    <string>Kick</string>
   </property>
  </action>
  <action name="actionSave_battle_record">
   <property name="text">
    <string>Save battle record</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
